### ISSUE TITLE
Feature Request:  I'd like it to be more obvious in the dock when an app is demanding attention

### ISSUE THREAD

**Issue by liquidonline** [psifidotos/Latte-Dock#460]
Hi,

I've been using latte-dock on a 27inch 2kHD monitor at home, and a 32inch 4kHD monitor at the office.  On both, but especially the larger one at work,  Sometimes, I am mildly annoyed at not being able to notice when there is an app requiring attention, and also to quickly take stock of what is open/closed/minimized. I tried moving them above the icons, and that has helped a little, but not a whole lot.   

I know the icon can "hop" temporarily, but it seems it isn't visible if the dock is hidden (I have it set to dodge active windows).  The secondary reason I care about this is because I noticed a significant resource drain when an app is asking for attention in latte-dock, and I ignore it.

I find there are already alot of knobs and switches to Latte-Dock, so I'm not going to suggest making it configurable.  My suggestion is to have the size of the dots be proportional to icon size.  Sometimes, I get the impression they are, and if so, maybe it just needs to be less of a linear relationship?  Maybe have it hop around longer, or have it hop around such that it's (partially) in view even when the dock is hidden?

**Comment by psifidotos** [comment]
For needs attentation the following are supported:

- the bounce lasts around 5 secs and this is a design decision to not be intrusive, i can set it maximum at 7secs
- when a task needs attention your dock should be raised for the previous 5 secs, if it doesnt it sounds like a bug in your system
- the high cpu comes from the color animatiom when a task needs attentation and I am in the process fixing it
- the design of lines and dots is one of the main characteristics of Latte and i have no intention changing it, it is already based on the icon size,

a dot, means a window is not active
a line, means a window is active

Latte provides two more options in Tasks page,
- Different colors for window states, this provides different color for minimized windows
- Dot indicator for an active window but also when it is a group of window

please next time for each issue/request open different threads in order to keep things organized...

**Comment by psifidotos** [comment]
@liquidonline many of the proposed fixes for needattention case I added them in our master branch...

concerning the showing of the dock when is hidden but a task needs attention, I am trying the following in my system and it works:

1. Use Dodge Active
2. open a dolphin window and make it overlap with the dock (this hides the dock)
3. apen a new dolphin window and immediately I click on the previous Dolphin window that overlaps with the dock (this makes the new created dolphin window to request needs attention)

In my system after (3) the dock is shown for 8,5secs with a dolphin window bouncing in it... the high cpu now is fixed as the color animation lasts only for these 8.5secs and at the same time I made the animation more light weight...
